src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -61,11 +61,17 @@
 register_cli_argument('policy', 'resource_group_name', arg_type=resource_group_name_type, help='the resource group where the policy will be applied')
 register_cli_argument('policy definition', 'policy_definition_name', arg_type=existing_policy_definition_name_type)
 register_cli_argument('policy definition create', 'name', options_list=('--name', '-n'), help='name of the new policy definition')
-register_cli_argument('policy definition', 'rules', help='JSON formatted string or a path to a file or uri with such content', type=file_type, completer=FilesCompleter())
+register_cli_argument('policy definition', 'rules', help='JSON formatted string or a path to a file with such content',
+                      type=file_type, completer=FilesCompleter())
```

Comment:
For these, you don't need to manually make these accept JSON or files and then sniff whether there is a file or not.  Instead, you can just make it accept JSON and remind people of the @syntax to load from a file.  So:

`Policy rule definition in JSON format. Use `@\<file\>` syntax to load from a file.`

Also, this seems like it may clobber a previous change that allowed this to come from a URI?

Reply:
Good catch - the help text was clobbered over. I'll update that.

For the @<file> recommendation, I made changes to 3 functions create_policy_definition, update_policy_definition and create_policy_assignment....I tried to follow the same existing pattern in each of them. Would you recommend updating to support @syntax in all places?